netci.groovy

```diff
@@ -828,6 +979,11 @@ def static getDockerImageName(def architecture, def os, def isBuild) {
                 return "hqueue/dotnetcore:ubuntu1404_cross_prereqs_v4-tizen_rootfs"
             }
         }
+        else if (architecture == 'arm') {
+            if (os == 'Ubuntu') {
+                return "microsoft/dotnet-buildtools-prereqs:ubuntu-14.04-cross-0cd4667-20170319080304"
```

Comment:
Let's update this to the last built image: microsoft/dotnet-buildtools-prereqs:ubuntu-16.04-cross-e435274-20180317125354

Reply:
done

Reply:
Sorry used 16.04 instead of 14.04 correct tag: microsoft/dotnet-buildtools-prereqs:ubuntu-14.04-cross-e435274-20180323032140